runtimes/web/src/ui/app.ts

```diff
@@ -76,6 +83,11 @@ export class App extends LitElement {
 
     private readonly diskPrefix: string;
 
+    private vibrateLevelIdx = 0;
```

Comment:
I wasn't sure where to store the vibration level. I tried in virtual-gamepad.ts at first, but didn't see how menu-overlay.ts could access it, so it ended up here.

Reply:
I'm defaulting to "OFF" for now, but if feedback is positive I recommend we default to "MEDIUM", I think the vibration really makes it feel more tactile.